### DISCUSSION TITLE
Thinking of getting a surface go 1st gen, what are some options for taking notes?

### DISCUSSION THREAD

**Discussion by Iheuzio** [linux-surface/linux-surface#1172] [Q&A] [answered]
As the title suggests, I am thinking of picking up a surface go 1st gen from 2018. It was obtained really cheap and offers what I need, which is taking notes. 

# Specifications

- 10-inch PixelSense display with a resolution of 1800 x 1200 with a pixel density of 217 PPI.
- Intel Pentium Gold Processor 4415Y and Intel HD Graphics 615 GPU.
- 64GB eMMC with 4GB ram.
- Can fully charge its battery in 2 hours.
- Comes with a headphone jack, a USB-C port, and a micro SD card slot.
- Wi-Fi version weighs starting at 1.15 lbs (522 g).

# Questions

1. **Does this support a specific pen model, without requiring external drivers? ie: 1776**

2. **What is the difference in battery life between windows and running it off a distro like fedora or ubuntu?**
 
3. **What are some good alternatives to OneNote?**
 
4. **Any other recommendations?**

**Comment by PhilDevProg** [comment, marked as answer]
1) I have the Surface Go 3 and it works with the normal Surface Pen without having to install additional drivers or userspace software like iptsd so I guess the Go 1 doesn't require something either.

2) In my experience (Fedora) battery life isn't great but okay on my Go 3 by default. But you can install auto-cpufreq which improved battery life very much for me.

3) I use Rnote which is great and has many features but there's also Xournal++.

**Reply by Iheuzio** [reply]
Thanks, I'll try it out when it arrives next week.
